dial_delay.go

```diff
@@ -0,0 +1,209 @@
+package swarm
+
+import (
+	"context"
+	"time"
+
+	ma "github.com/multiformats/go-multiaddr"
+	mafmt "github.com/whyrusleeping/mafmt"
+)
+
+const p_circuit = 290
```

Comment:
we should probably capitalize this to `P_CIRCUIT` -- that's what we do everywhere else.